dotnet-desktop-guide/net/winforms/controls-design/overview.md

```diff
@@ -56,61 +72,44 @@ Deciding which kind of control to create can be confusing. This article highligh
             <li>You need to implement custom functionality that isn't available through standard controls.</li>
         </ul>
     </td>
-    <td><a href="#custom-controls">Custom control</a> by inheriting from <a href="/dotnet/api/system.windows.forms.control">System.Windows.Forms.Control</a>.</td>
+    <td>Create a <a href="#custom-controls">custom control</a> by inheriting from <a href="/dotnet/api/system.windows.forms.control">System.Windows.Forms.Control</a>.</td>
   </tr>
 </tbody>
 </table>
 
-## Base Control Class
+## User controls
 
-The <xref:System.Windows.Forms.Control> class is the base class for Windows Forms controls. It provides the infrastructure required for visual display in Windows Forms applications and provides the following capabilities:
+A user control is a collection of Windows Forms controls presented as a single control to the consumer. This kind of control is referred to as a _composite control_. The contained controls are called _constituent controls_.
 
-- Exposes a window handle.
-- Manages message routing.
-- Provides mouse and keyboard events, and many other user interface events.
-- Provides advanced layout features.
-- Contains many properties specific to visual display, such as <xref:System.Windows.Forms.Control.ForeColor%2A>, <xref:System.Windows.Forms.Control.BackColor%2A>, <xref:System.Windows.Forms.Control.Height%2A>, and <xref:System.Windows.Forms.Control.Width%2A>.
-- Provides the security and threading support necessary for a Windows Forms control to act as a Microsoft® ActiveX® control.
-
-Because so much of the infrastructure is provided by the base class, it's relatively easy to develop your own Windows Forms controls.
+A user control holds all of the inherent functionality associated with each of the contained Windows Forms controls and enables you to selectively expose and bind their properties. A user control also provides a great deal of default keyboard handling functionality with no extra development effort on your part.
 
-## Composite Controls
+For example, a user control could be built to display customer address data from a database. This control would include a <xref:System.Windows.Forms.DataGridView> control to display the database fields, a <xref:System.Windows.Forms.BindingSource> to handle binding to a data source, and a <xref:System.Windows.Forms.BindingNavigator> control to move through the records. You could selectively expose data binding properties, and you could package and reuse the entire control from application to application.<!-- TODO For an example of this kind of user control, see [How to: Apply Attributes in Windows Forms Controls](how-to-apply-attributes-in-windows-forms-controls.md).-->
 
-A composite control is a collection of Windows Forms controls encapsulated in a common container. This kind of control is sometimes called a *user control*. The contained controls are called *constituent controls*.
```

Comment:
I always (perhaps incorrectly) think of user controls as controls not provided by the winforms library, but ones that are created by a user.

When you compare the wording in the red (deleted) and green (added), what I'm really doing is swapping how it's talked about. It makes more sense to me to do it this way. The deleted content was presenting the control information as a "composite control" that is some times referred to as a "user control" but since, as you pointed out, the base class is `UserControl` it makes sense to me to talk about it in reverse. It's a "user control" that you can refer to as a "composite control."

Reply:
Yes, I agree!